Formula/q/qt.rb

```diff
@@ -187,63 +183,73 @@ def install
-      # https://bugreports.qt.io/browse/QTBUG-115357
-      cmake_args << "-DQT_FEATURE_webengine_system_libpng=OFF"
-
       cmake_args << "-DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}.0"
-      config_args << "-sysroot" << MacOS.sdk_path.to_s
+
+      config_args += %W[
+        -sysroot #{MacOS.sdk_path}
+        -no-feature-ffmpeg
+      ]
+      # FIXME: Workaround for error: use of undeclared identifier 'importMemoryHandleInfo'
+      # Remove once properly handled by Qt.
+      config_args << "-no-feature-webengine-vulkan"
       # NOTE: `chromium` should be built with the latest SDK because it uses
       # `___builtin_available` to ensure compatibility.
-      config_args << "-skip" << "qtwebengine" if DevelopmentTools.clang_build_version <= 1200
+      config_args += %w[-skip qtwebengine] if DevelopmentTools.clang_build_version <= 1200
     else
       # Explicitly specify QT_BUILD_INTERNALS_RELOCATABLE_INSTALL_PREFIX so
       # that cmake does not think $HOMEBREW_PREFIX/lib is the install prefix.
       cmake_args << "-DQT_BUILD_INTERNALS_RELOCATABLE_INSTALL_PREFIX=#{prefix}"
 
-      # The vendored copy of libjpeg is also used instead of the brewed copy, because the build
-      # fails due to a missing symbol otherwise.
-      # On macOS chromium will always use bundled copies and the QT_FEATURE_webengine_system_*
-      # arguments are ignored.
-      cmake_args += %w[
-        -DQT_FEATURE_webengine_system_alsa=ON
-        -DQT_FEATURE_webengine_system_ffmpeg=ON
-        -DQT_FEATURE_webengine_system_icu=ON
-        -DQT_FEATURE_webengine_system_libevent=ON
-        -DQT_FEATURE_webengine_system_libpng=ON
-        -DQT_FEATURE_webengine_system_libxml=ON
-        -DQT_FEATURE_webengine_system_libwebp=ON
-        -DQT_FEATURE_webengine_system_minizip=ON
-        -DQT_FEATURE_webengine_system_opus=ON
-        -DQT_FEATURE_webengine_system_poppler=ON
-        -DQT_FEATURE_webengine_system_pulseaudio=ON
-        -DQT_FEATURE_webengine_system_zlib=ON
+      # NOTE: for QtWebEngine arguments:
+      # * The vendored copy of `libjpeg` is used instead of the brewed copy,
+      #   because the build fails due to a missing symbol otherwise.
+      # * The vendored copy of `re2` is used to avoid rebuilds with `re2` version
+      #   bumps and due to frequent API incompatibilities in Qt's copy of Chromium
+      # * On macOS Chromium will always use bundled copies and the -system-webengine-*
+      #   arguments (and matching QT_FEATURE_webengine_system_* variables) are ignored.
+      # * As of Qt 6.6.0, webengine-ozone-x11 feature appears to be mandatory for Linux.
+      config_args += %w[
+        -feature-webengine-ozone-x11
+        -system-webengine-alsa
+        -system-webengine-ffmpeg
+        -system-webengine-freetype
+        -system-webengine-harfbuzz
+        -system-webengine-icu
+        -system-webengine-lcms2
+        -system-webengine-libevent
+        -system-webengine-libpng
+        -system-webengine-libvpx
```

Comment:
And probably remove this. I added it after copying all possible system lib, but noticed Arch ref https://codereview.qt-project.org/c/qt/qtwebengine/+/454908